dashboard/backend/create-handler.ts

```
@@ -0,0 +1,250 @@
+// import { jwtVerify } from "jose/jwt/verify";
+// import { JWK } from "jose";
+
+export const CORS = {
+  "Access-Control-Allow-Origin": "*",
+  "Access-Control-Allow-Methods": "GET,POST,OPTIONS,PUT,DELETE",
+  "Access-Control-Allow-Headers": "Origin, Content-Type, Accept",
+  "Access-Control-Max-Age": "86400",
+};
+
+interface ClerkTemplate {
+  readonly app_metadata: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)